src/index.html

```diff
@@ -11,7 +11,188 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        
+        <div>
+          <label for="surname">Surname:</label>
+          <input 
+            type="text" 
+            name="Surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="container">
+          <label for="name">Name:</label>
+          <input 
+            type="text" 
+            name="Name" 
+            id="name"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="container">
+          <label for="age">How old are You?</label>
+          <input 
+          type="number" 
+          name="Age" 
+          id="age" 
+          min="1"
+          max="100"
+          value="12"
+          >
+        </div>
+
+        <div class="container">
+          <label for="birthday">Full date of birth:</label>
+          <input type="date" placeholder="mm/dd/yyyy">
+        </div>
+
+        <div class="container">
+          <label for="terms">
+            I accept the term of the agreement
+          </label>
+          <input type="checkbox">
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fields">
+
+        <legend>Registration</legend>
```

Comment:
No needs to add empty line between parent and child
```suggestion
      <fieldset class="fields">
        <legend>Registration</legend>
```